Submitted By: Jim Gifford (patches at jg555 dot com)
Date: 2003-06-03
Initial Package Version: 1.4.3
Origin: Jim Gifford
Description: Fixes GCC 3.3 Compile Errors

$LastChangedBy: $
$Date: $

diff -Naur svgalib-1.4.3.orig/src/vga.c svgalib-1.4.3/src/vga.c
--- svgalib-1.4.3.orig/src/vga.c	2001-05-29 07:37:55.000000000 -0000
+++ svgalib-1.4.3/src/vga.c	2003-06-03 22:31:55.000000000 -0000
@@ -3914,7 +3914,7 @@
 
 #define ML_GETINT(x) \
 	ptr = strtok(NULL, " "); if(!ptr) break; \
-	mmt.##x = atoi(ptr);
+	mmt.x = atoi(ptr);
 
 	    ML_GETINT(HDisplay);
 	    ML_GETINT(HSyncStart);
